### PR TITLE
Trogg and Drogbar religion

### LINKED ISSUE #148: Adding New Broken Isles Races, Religions and Culture for mapping
To prepare the mapping, I think that we should already add the races, religions and cultures of the Broken Isles.

**Vrykuls:**
 - [x] _Culture_: Stormir (To give a different culture to the Vrykul here than to those in Northrend. They are separated, so it is normal that they have different cultures.) Added
 
**Nightborne:**
- [x] _Religion:_ Shalarism (Cult of Nightwell) #222/#309 Merged
- [x] _Culture:_ Shalassian (from their language) Added
- [x] _Race trait:_ Nightborne/Shal'dorei #222/#291 #291 Merged

**Highmountain Taurens**
- [x] _Culture:_ Highmountain (tauren group) Added
- [x] _Race trait_: Does they need specific race trait with blessing of Eche'ro ?  #306 Merged

**Drogbar**
- [ ] _Religion:_ I don't know their religion but they seem to revere things related to Deathwing. So probably we can create a dragon religion to worship Deathwing, which would also be the religion of the black dragons. #344
- [x] _Culture:_ Drogbar (Trogg group) #295 Merged
- [x] _Race trait:_ Drogbar  #295 Merged

### LINKED ISSUE #330: Faiths for Trogg/Drogbar
**Describe your suggestion in full detail below:**
Faiths for Trogg/Drogbar. They worship animals. Troggs worship the massive reptiles known as salamanders. Drogbar worship giant worms like Naraxas.

### PR DESCRIPTION
**Pull request by robmart**
Resolves #330, resolves  #148

## Changelog:
- Added Trogg religion
- Added Drogbar religion

# TODO:
- ~~Trogg religion loc ~~(Need a name suggestion)~~ (Need a description)~~
- ~~Trogg religion icon (something based on a salamander/lizard)~~ Zumbak the wonderful made an icon :succ_kiss:

# How to test:

### CODE REVIEW

File: localization/english/religion/wc_religion_l_english.yml [hunk 1]
```diff
@@ -641,4 +641,16 @@
...
  cult_of_nightwell_adherent_plural:0 "Shal'danalists"
- cult_of_nightwell_desc:0 "Shal'danalism are the Nightwell traditions. Its followers depend on and are changed by the Nighwell's power, the Shal'danalists favor the arcane philosophy of their past as they preserve and evolve their ways in their unending isolation from the outside world."
\ No newline at end of file
+ cult_of_nightwell_desc:0 "Shal'danalism are the Nightwell traditions. Its followers depend on and are changed by the Nighwell's power, the Shal'danalists favor the arcane philosophy of their past as they preserve and evolve their ways in their unending isolation from the outside world."
+
+ trogg_religion:0 "Trogg Religion" #Placeholder
+ trogg_religion_adj:0 "Shal'danalic"
+ trogg_religion_adherent:0 "Shal'danalist"
+ trogg_religion_adherent_plural:0 "Shal'danalists"
+ trogg_religion_desc:0 "Shal'danalism are the Nightwell traditions. Its followers depend on and are changed by the Nighwell's power, the Shal'danalists favor the arcane philosophy of their past as they preserve and evolve their ways in their unending isolation from the outside world."
```
Comment (zumbak04) [inline localization/english/religion/wc_religion_l_english.yml:650]: A name idea is here. The description is to be written by some English Proofreader, ping them in the dev.
```suggestion
 trogg_religion:0 "Salamandrism" #Placeholder
 trogg_religion_adj:0 "Salamandric"
 trogg_religion_adherent:0 "Salamandrist"
 trogg_religion_adherent_plural:0 "Salamandrists"
 trogg_religion_desc:0 "We smash and worship salamanders (placeholder)."
```
Source: https://wowpedia.fandom.com/wiki/Trogg#Culture

File: common/religion/religions/shamanism_group.txt [hunk 2]
```diff
@@ -619,5 +619,68 @@
...
+
+			#Main Group
+			doctrine = doctrine_no_head
+
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_ancestor_worship
+
+			#Magic
+			doctrine = doctrine_elemental_earth_magic_approved #Drogbar have strong control over earth
```
Comment (zumbak04) [inline common/religion/religions/shamanism_group.txt:683]: IIRC, you can't have 2 approved doctrines. So one must go.
<details>
<summary>Click to expand</summary>

image

</details>

Comment (zumbak04) [inline common/religion/religions/shamanism_group.txt:683]: I would say that the drogbar wouldn't mind the Decay.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/religions/shamanism_group.txt [hunk 3]
```diff
@@ -619,5 +619,68 @@
...
+
+			#Main Group
+			doctrine = doctrine_no_head
+
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_human_sacrifice
+
+			#Magic
+			#Ignore all except some shaman, they don't know any other magic
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply common/religion/religions/shamanism_group.txt:648]: This guy can into Fire magic too.

File: common/religion/religions/shamanism_group.txt [hunk 4]
```diff
@@ -619,5 +619,67 @@
...
+
+			#Main Group
+			doctrine = doctrine_no_head
+
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_ancestor_worship
+
+			#Magic
+			doctrine = doctrine_elemental_earth_magic_approved #Drogbar have strong control over earth
```
Comment (zumbak04) [inline common/religion/religions/shamanism_group.txt:682]: I propose these laws for both faiths:
Shadow - Ignored or Shunned, they're primitive and don't care for bad magic use
Fel - Ignored or Shunned, they're primitive and don't care for magic use
Death - Ignored or Shunned, they're primitive and don't care for magic use
Decay - Approved, they don't worship elements thus don't mind enslaving them
Fire - Accepted, troggs use it and they live close to lava and stuff.
Air - Ignored, their counter-magic.
Earth - Idolised... well, you know.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/holy_sites/wc_holy_sites.txt [hunk 5]
```diff
@@ -1180,6 +1180,17 @@ taunkale = {
...
 #grizzlemaw
 
+### Trogg religion
+alterac_valley = {
+	county = c_alterac_valley
+
+	character_modifier = {
+		name = holy_site_alterac_valley_effect_name
+		martial = 1
+		monthly_prestige_gain_per_knight_add = 0.1
```
Comment (zumbak04) [inline common/religion/holy_sites/wc_holy_sites.txt:1190]: Balanced it [Here](https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit?usp=sharing).
```suggestion
		martial = 1
		monthly_prestige_gain_per_knight_add = 0.2
```

File: common/religion/religions/shamanism_group.txt [hunk 6]
```diff
@@ -619,5 +619,71 @@
...
+			doctrine = tenet_natural_primitivism
+			doctrine = tenet_sanctity_of_nature
+			doctrine = tenet_ancestor_worship
+
+			#Magic
+			doctrine = doctrine_shadow_magic_ignored # Some Drogbar are part of Twilight Hammer
+			doctrine = doctrine_elemental_decay_magic_accepted #Drogbar would probably not mind enslaving the elements
+			doctrine = doctrine_elemental_water_magic_ignored
+			doctrine = doctrine_elemental_air_magic_ignored
+			doctrine = doctrine_elemental_earth_magic_approved #Drogbar have strong control over earth
```
Comment (zumbak04) [inline common/religion/religions/shamanism_group.txt:686]: Only one magic can be Idolised.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/holy_sites/wc_holy_sites.txt [hunk 7]
```diff
@@ -1180,6 +1180,17 @@ taunkale = {
 #njord
 #grizzlemaw
 
+### Trogg religion
+alterac_valley = {
```
Comment (zumbak04) [inline common/religion/holy_sites/wc_holy_sites.txt:1184]: Not localized.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by hamjustine — APPROVED** [review]

**Review by arithon — APPROVED** [review]
loc LGTM

**Pull request merged**